src/ppom-controller.ts

```diff
@@ -414,6 +419,28 @@ export class PPOMController extends BaseControllerV2<
     this.#fileScheduleInterval = undefined;
   }
 
+  #setToActiveState() {
+    this.messagingSystem.publish(
+      'PPOMController:initialisationStateChangeEvent',
+      PPOMInitialisationStatus.INPROGRESS,
+    );
+    this.#reinitPPOMForChainIfRequired(ETHEREUM_CHAIN_ID)
+      .then(async () => {
+        this.messagingSystem.publish(
+          'PPOMController:initialisationStateChangeEvent',
+          PPOMInitialisationStatus.SUCCESS,
+        );
+        this.#checkScheduleFileDownloadForAllChains();
+      })
+      .catch((error: Error) => {
+        this.messagingSystem.publish(
+          'PPOMController:initialisationStateChangeEvent',
+          PPOMInitialisationStatus.FAIL,
+        );
+        console.error(`Error in initialising ppom: ${error.message}`);
```

Comment:
does PPOM have access to sentry? would it help to capture these errors?

Reply:
I do not think controllers have direct access to sentry, error from controllers need to be propagated.